static/scripts/help.js

```diff
@@ -1,7 +1,93 @@
+import livesearch from "./helpers/livesearch.js";
+import "./help/contactForm.js";
+
+
 $(document).ready(function () {
     $('.btn-poll').on('click', function (e) {
         e.preventDefault();
 
         document.cookie = "pollClicked=true; expires=Fri, 31 Dec 9999 23:59:59 GMT; path=/";
     });
 });
+
+// iFrame full height
+document.querySelectorAll("iframe").forEach((iframe, index)=>{
+    const identifier = "iframe-"+index;
+    iframe.dataset.identifier = identifier;
+    iframe.addEventListener("load", (event) => {
+        iframe.contentWindow.postMessage(JSON.stringify({
+            function: "getSize",
+            identifier: identifier
+        }), '*');
+    });
+});
+
+window.addEventListener("message", (event) => {
+    const message = JSON.parse(event.data);
+    if(message.size && message.identifier){
+        document.querySelectorAll(`iframe[data-identifier="${message.identifier}"]`).forEach((iframe) => {
+            iframe.style.height = message.size.y + 'px';
+        });
+        console.log("remove!");
+        document.querySelectorAll(".mobile-warning").forEach((warning) => {
+            warning.remove();
+        });
+    }
+}, false);
+
+/* script for iFrame
+window.addEventListener("message", (event) => {
+    const message = JSON.parse(event.data);
+    if(message.function){
+        if(message.function === "getSize" && message.identifier){
+            window.parent.postMessage(JSON.stringify({
+                identifier: message.identifier,
+                size: {
+                    x: document.body.scrollWidth,
+                    y: document.body.scrollHeight
+                }
+            }), '*');
+        }
+    }
+}, false);
+*/
```

Comment:
same here, wird der auskommentierte Code noch benötigt?

Reply:
Das ist der Code welcher in Confluence eingefügt werden muss, damit die gesamte Seite funktioniert. Muss ich scheinbar noch besser kommentieren. Oder haben wir einen Ort wo wir solchen Code speichern können? Nur auf irgendein komisches input Feld bei Confluence möchte ich mich da echt nicht verlassen. Falls das doch mal abstürtzt oder so weiß ich nicht ob sowas mit im BackUp ist.